packages/form/src/components/SelectInputFieldV2/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import type { Meta } from '@storybook/react'
+import { Snippet, Stack, Text } from '@ultraviolet/ui'
+import { useForm } from 'react-hook-form'
+import { Form, SelectInputFieldV2 } from '../..'
+import { mockErrors } from '../../../mocks'
+
+export default {
+  component: SelectInputFieldV2,
+  decorators: [
+    ChildStory => {
+      const methods = useForm()
+      const {
+        errors,
+        isDirty,
+        isSubmitting,
+        touchedFields,
+        submitCount,
+        dirtyFields,
+        isValid,
+        isLoading,
+        isSubmitted,
+        isValidating,
+        isSubmitSuccessful,
+      } = methods.formState
+
+      return (
+        <Form
+          onRawSubmit={() => {}}
+          errors={mockErrors}
+          methods={methods}
+          name="SelectInput"
+        >
+          <Stack gap={2}>
+            <ChildStory />
+            <Stack gap={1}>
+              <Text variant="bodyStrong" as="p">
+                Form input values:
+              </Text>
+              <Snippet prefix="lines" initiallyExpanded>
+                {JSON.stringify(methods.watch(), null, 1)}
+              </Snippet>
+            </Stack>
+            <Stack gap={1}>
+              <Text variant="bodyStrong" as="p">
+                Form values:
+              </Text>
+              <Snippet prefix="lines">
+                {JSON.stringify(
+                  {
+                    errors,
+                    isDirty,
+                    isSubmitting,
+                    touchedFields,
+                    submitCount,
+                    dirtyFields,
+                    isValid,
+                    isLoading,
+                    isSubmitted,
+                    isValidating,
+                    isSubmitSuccessful,
```

Comment:
Just asking, spreading props `...methods` for this case is forbidden ? Better to always expose props tho